test/run

```diff
@@ -118,7 +118,7 @@ scl_usage() {
 }
 
 function test_scl_usage() {
-  scl_usage "ruby --version" "ruby ${RUBY_VERSION}."
+  scl_usage "ruby --version" "ruby ${VERSION}."
```

Comment:
$VERSION should be se to empty string. Not sure this test does what it should have.

Reply:
The test tests if ruby version is the same as tests. Like we do in the rest containers.
Nevertless, RUBY_VERSION is specified in container and not in host system where the test is running.

As examples see:
https://github.com/sclorg/s2i-php-container/blob/master/test/run#L186
https://github.com/sclorg/s2i-nodejs-container/blob/master/test/test-lib-nodejs.sh#L250